src/commands/utils/clusters.ts

```diff
@@ -447,6 +447,20 @@ export async function deleteCluster(
     }
 }
 
+export async function abortLastOperationInCluster(
+    target: AksClusterTreeItem,
+    clusterName: string
+): Promise<Errorable<string>> {
+    try {
+        const containerClient = getContainerClient(target);
+        await containerClient.managedClusters.beginAbortLatestOperationAndWait(target.resourceGroupName, clusterName)
```

Comment:
What happens here if there's no operation in progress? Is it a no-op, or an error?

And is there a way to check if an operation is in progress? (Noting that attempting to check in advance might lead to race conditions.)

Reply:
yeah this happens :)  basically fairly detail in the error. 

I tried this in order and vice-versa - but I am guessing just like stop/start if mis-used then it will cause issue in cluster :P  you know what I mean.

<img width="547" alt="Screenshot 2023-10-06 at 10 46 33 AM" src="https://github.com/Azure/vscode-aks-tools/assets/6233295/64af164e-88ef-4e23-af48-8cb93413853b">

Reply:
I wonder if we can check the cluster provisioning state before asking the user "Do you want to abort last operation in cluster ${clusterName}?"...

If it's not in a suitable state, we could just pop up an info message saying that?

Reply:
Nice idea lemme update that :) thank you.